### ISSUE TITLE
Flags require capitalization? Bug or feature?

### ISSUE THREAD

**Issue by ieugen** [lambdaisland/cli#5]
Hi,

I encountered this during my cli implementation. 
This is not specified anywhere in the documentation. 

Using the bellow 
```clojure
:flags ["--name string" "A name option - lower case `string` does not get parsed"]

;; gets me
[FATAL] Unknown flag:  --name
```

Using capitalized version works ok
```clojure
:flags ["--name STRING" "A name option - lower case `string` does not get parsed"]
```

**Comment by plexus** [comment]
Yes, this is by design. This follows the connections I've seen "in the wild", this is the "classic" way of doing things, in all caps, or we also support the "modern" way, eg `--name <string>`.

A PR for the README to point that out would be most welcome.

**Comment by plexus** [comment]
The README has been updated.
